packages/components/src/components/buttons/demos/buttons.demo.tsx

```diff
@@ -0,0 +1,78 @@
+import { Component, h, Host, Fragment } from '@stencil/core';
+import type { ButtonVariant } from '../types';
+import { ES_COMPONENTS_ICON_NAMESPACE, type IconDescription } from '../../..';
+
+/** Basic button demon. */
+@Component({
+    tag: 'es-buttons-demo',
+    styleUrl: './buttons-demo.css',
+    shadow: true,
+})
+export class LoadingTextDemo {
+    private variants: ButtonVariant[] = [
+        'default',
+        'filled',
+        'outline',
+        'delete',
+        'cancel',
+        'minimal',
+        'link',
+    ];
+
+    render() {
+        return (
+            <Host>
+                {this.variants.map((variant) => (
+                    <>
```

Comment:
For my future reference: No 'key' needed here because this list is static and Stencil doesn't use 'key' for its diffing process.